### DISCUSSION TITLE
M5StickC Connected but not showing up

### DISCUSSION THREAD

**Discussion by antoniowas** [josephdadams/TallyArbiter#439] [General]
Hey guys, I've spent several hours trying to figure this out. So thought I'd reach out.
I've flashed the M5StickC using the latest .bin file (and a few older ones, for all that matters), and my serial monitor is telling me that it's successfully connected to the TA server. But it doesn't show up on the connected listeners list. 

![image](https://user-images.githubusercontent.com/40467639/167122827-b569e0c1-4e0d-4efd-9b76-b97908dff9e3.png)

Here is a screenshot from the serial monitor. 
Now a screenshot from the TA window. 

![image](https://user-images.githubusercontent.com/40467639/167122946-0af93d95-4e29-4a1d-9ff9-5c665123395e.png)

And here's what the M5 looks like
![image](https://user-images.githubusercontent.com/40467639/167123093-1660f428-62b4-421c-870f-82cf17d2cd80.png)

I'm really confused at this point. Tried a bunch of different things, leading to the same result. Rebooted the server, reflashed M5. 

Anyone has an idea of what might be going on here? 

The TA is working with other web interfaces, like laptops and phones. The issue seems to be with the M5 connection. 

Thanks a lot in advance guys.

**Comment by hrueger** [comment]
> I've flashed the M5StickC using the latest .bin file

where did you get it from? The firmware here in this monorepo is only compatible with TallyArbiter Version 3 (beta).

It looks like you're running TA 2?

**Reply by antoniowas** [reply]
Oh wow. I must have missed that. 
I was banging my head against the wall full on. 
Text is still kind of chopped off in the M5Stick, where it shows TA server info and battery.
Other than that, it works as it should. Thanks for taking a look for your time. 
Cheers,
Antonio
